src/flp/szk.rs

```diff
@@ -221,7 +435,17 @@ where
         self.typ.joint_rand_len() > 0
     }
 
-    fn prove(
+    pub(crate) fn proof_len(&self) -> usize {
+        self.typ.proof_len()
+    }
+
+    /// Used by a client to prove validity (according to an FLP system) of an input
+    /// that is both shared between the leader and helper
+    /// and encoded as a measurement. Has a precondition that leader_input_share
+    /// + helper_input_share = encoded_measurement.
+    ///     leader_seed_opt should be set only if the underlying FLP system requires
+    ///     joint randomness.
```

Comment:
```suggestion
    /// leader_seed_opt should be set only if the underlying FLP system requires
    /// joint randomness.
```

Reply:
I'd also clarify that, if the helper's seed is reused for blinding the joint randomness computation.